_posts/2016-08-27-mouse-dna-magnetic-fields.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: Low magnetic fields leave mouse DNA undamaged
+author: RaoOfPhysics
+image:
+  feature: 2016-08-27-mouse-dna-magnetic-fields.jpg
+  credit: Nogas1974
+  caption: “DNA com GGN”
+---
+
+_Cells in brain, kidney and liver shown to withstand sustained exposure to magnetic fields._
```

Comment:
noticed that we haven't used periods at the end of the dek. want to remove this one? @RaoOfPhysics